src/js/module.js

```diff
@@ -204,18 +202,36 @@ iotjs_module_t.load = function(id, parent) {
   return module.exports;
 };
 
+iotjs_module_t.loadRemote = function(id, parent) {
+  if (process.builtin_modules[id[0]]) {
```

Comment:
Can this case occur at all?

Reply:
Honestly, I neither do I think it would happen.